tests/components/zha/test_config_flow.py

```diff
@@ -134,60 +146,55 @@ async def test_probe_radios(hass):
+        context={CONF_SOURCE: "pick_radio"},
+        data={CONF_RADIO_TYPE: RadioType.ezsp.description},
     )
 
-    with patch.dict(config_flow.RADIO_TYPES, {"ezsp": {CONTROLLER: app_ctrl_cls}}):
-        result = await hass.config_entries.flow.async_configure(
-            result["flow_id"],
-            user_input={zigpy.config.CONF_DEVICE_PATH: "/dev/ttyUSB33"},
-        )
-        assert result["type"] == RESULT_TYPE_FORM
-        assert result["step_id"] == "port_config"
-        assert result["errors"]["base"] == "cannot_connect"
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"], user_input={zigpy.config.CONF_DEVICE_PATH: "/dev/ttyUSB33"},
+    )
+    assert result["type"] == RESULT_TYPE_FORM
+    assert result["step_id"] == "port_config"
+    assert result["errors"]["base"] == "cannot_connect"
+    assert probe_mock.await_count == 1
 
 
-@pytest.mark.parametrize(
-    "radio_type, orig_ctrl_cls",
-    ((name, r[CONTROLLER]) for name, r in RADIO_TYPES.items()),
-)
 @patch("homeassistant.components.zha.async_setup_entry", AsyncMock(return_value=True))
-async def test_user_port_config(hass, radio_type, orig_ctrl_cls):
+@patch("bellows.zigbee.application.ControllerApplication.probe", return_value=True)
+async def test_user_port_config(probe_mock, hass):
     """Test port config."""
-    app_ctrl_cls = MagicMock()
-    app_ctrl_cls.SCHEMA_DEVICE = orig_ctrl_cls.SCHEMA_DEVICE
-    app_ctrl_cls.probe = AsyncMock(return_value=True)
     await setup.async_setup_component(hass, "persistent_notification", {})
 
     result = await hass.config_entries.flow.async_init(
-        DOMAIN, context={CONF_SOURCE: "pick_radio"}, data={CONF_RADIO_TYPE: radio_type}
+        DOMAIN,
+        context={CONF_SOURCE: "pick_radio"},
+        data={CONF_RADIO_TYPE: RadioType.ezsp.description},
     )
 
-    with patch.dict(
-        config_flow.RADIO_TYPES,
-        {radio_type: {CONTROLLER: app_ctrl_cls, "radio_description": "radio"}},
-    ):
-        result = await hass.config_entries.flow.async_configure(
-            result["flow_id"],
-            user_input={zigpy.config.CONF_DEVICE_PATH: "/dev/ttyUSB33"},
-        )
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"], user_input={zigpy.config.CONF_DEVICE_PATH: "/dev/ttyUSB33"},
+    )
 
-        assert result["type"] == "create_entry"
-        assert result["title"].startswith("/dev/ttyUSB33")
-        assert (
-            result["data"][zigpy.config.CONF_DEVICE][zigpy.config.CONF_DEVICE_PATH]
-            == "/dev/ttyUSB33"
-        )
-        assert result["data"][CONF_RADIO_TYPE] == radio_type
+    assert result["type"] == "create_entry"
+    assert result["title"].startswith("/dev/ttyUSB33")
+    assert (
+        result["data"][zigpy.config.CONF_DEVICE][zigpy.config.CONF_DEVICE_PATH]
+        == "/dev/ttyUSB33"
+    )
+    assert result["data"][CONF_RADIO_TYPE] == "ezsp"
+    assert probe_mock.await_count == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)